<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatternEditor</class>
 <widget class="QDialog" name="PatternEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&amp;Pattern</string>
        </property>
        <property name="buddy">
         <cstring>cbPattern</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbPattern">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bnSavePattern">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>96</width>
          <height>17</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="bnNewPattern">
        <property name="text">
         <string>New pattern</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>&amp;Description code:</string>
        </property>
        <property name="buddy">
         <cstring>edDescription</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0" rowspan="3">
       <widget class="QTextEdit" name="edDescription"/>
      </item>
      <item row="3" column="2" rowspan="3" colspan="3">
       <widget class="QListWidget" name="listSynonyms">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>&amp;Guard code:</string>
        </property>
        <property name="buddy">
         <cstring>edGuard</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="0" rowspan="2">
       <widget class="QTextEdit" name="edGuard"/>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_4">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>13</height>
         </size>
        </property>
        <property name="text">
         <string>&amp;Synonyms:</string>
        </property>
        <property name="buddy">
         <cstring>listSynonyms</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>&amp;Name</string>
        </property>
        <property name="buddy">
         <cstring>edPatternName</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="edPatternName">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QWidget" name="widget" native="true">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <widget class="QPushButton" name="bnAddSynonym">
         <property name="geometry">
          <rect>
           <x>31</x>
           <y>4</y>
           <width>30</width>
           <height>23</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>+</string>
         </property>
        </widget>
        <widget class="QPushButton" name="bnRemSynonym">
         <property name="geometry">
          <rect>
           <x>89</x>
           <y>4</y>
           <width>30</width>
           <height>23</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </widget>
      </item>
      <item row="0" column="1" rowspan="7">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cbPattern</tabstop>
  <tabstop>edDescription</tabstop>
  <tabstop>edGuard</tabstop>
  <tabstop>bnSavePattern</tabstop>
  <tabstop>bnNewPattern</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
